content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -62,6 +62,11 @@ and respecting the PodDisruptionBudget you have defined). It is then safe to
 bring down the node by powering down its physical machine or, if running on a
 cloud platform, deleting its virtual machine.
 
+{{< note >}}
+[nodeName](/docs/concepts/scheduling-eviction/assign-pod-node/#nodename) bypasses the scheduler, 
+thus evicted Pods will still run on a drained node.
```

Comment:
"will still run" is a little bit tricky to parse ...
These Pods are terminated anyway, and then restarted?
Or, these Pods are not terminated at all, due to the `nodeName` specified?

Reply:
@sftim suggestion describes it better (thank you by the way)

Reply:
This conversation can be closed. More feedbacks added to the revised text because I'm still curious what is happening behind the scene.